server/updater/errors.go

```diff
@@ -0,0 +1,50 @@
+package updater
+
+import (
+	"errors"
+	"fmt"
+)
+
+var (
+	// ErrNoNewerVersion error is returned when plugin has no a newer version on the Marketplace.
+	ErrNoNewerVersion = errors.New("plugin has no a newer version")
+
+	// ErrPluginInSkipList error is returned when plugin is in the skip plugins list.
+	ErrPluginInSkipList = errors.New("plugin restricted to be installed, it is in the skip list")
+
+	// ErrDifferentPlugins error is returned when two plugins are not the same by their id.
+	ErrDifferentPlugins = errors.New("plugins are not the same, it cannot be updated")
+)
+
+// ServerVersionError is returned when new version of a plugin is not compatible
```

Comment:
I'd suggest skipping the modelling of this -- in practice, the marketplace will never return an update to a server version for which the plugin is not compatible.